### DISCUSSION TITLE
Turn away from flash.

### DISCUSSION THREAD

**Discussion by AuG-24** [danielkrupinski/Osiris#3305] [Ideas]
As per my understanding , flash reduction exists in osiris but using it would look super blatant.. and here comes the OW ban.. My idea is to implement a "quick turn away from flash" where based on the trajectory of the flash , our POV turns away from it and then returns to as it was , after  the flash pops. This would benefit players trying to play legit.

**Comment by aoqia194** [comment]
Or you could actually just turn away from the flash (/s).
I doubt it would be hard to move your character's yaw +180 degrees and then back, with smoothing obviously.
